### ISSUE TITLE
Opaque region is not scaled to surface coordinates

### ISSUE THREAD

**Issue by tomKPZ** [hyprwm/Hyprland#1458]
The opaque region is an optimization to prevent drawing underneath opaque parts of windows.  Hyprland seems to respect the hint, but treats the region in px coordinates instead of surface coordinates.  This causes rendering artifacts when using 2x scaling which can be seen in eg. Chromium when using a transparent tabstrip.  This issue doesn't occur in sway.

In the below screenshot, you can see Chromium is in front of a white region, but only the upper-left quadrant of the tabstrip is showing white.

![20230130_15h39m07s_grim](https://user-images.githubusercontent.com/9124077/215621672-1a0455cb-bc31-4c74-92b5-3d2520de9bf3.png)

**Comment by tomKPZ** [comment]
1. Add the following to `~/.config/gtk-3.0/gtk.css`
```
headerbar.chromium {
  background-image: none;
  background-color: rgba(0, 0, 0, 0.5);
}
```
3. You must be running a dev version of Chrome since this [feature was added recently](https://chromium-review.googlesource.com/c/chromium/src/+/4128765).  On Arch Linux, I installed google-chrome-dev from the AUR and launch with google-chrome-unstable.
4. Enable the GTK theme in chrome://settings/appearance
5. Right click on the tabstrip background and make sure "Use system titlebar and borders" is unselected

Happy to help further with getting it working.
